src/styles/_catalog.scss

```diff
@@ -1,22 +1,40 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($card-width, 1fr));
+  gap: 48px;
 
```

Comment:
The layout a little bit differ from the one built using the display: flex.
Probably, you need to add justify-content: center.